### ISSUE TITLE
NVDA is very silient on VS Code

### ISSUE THREAD

**Issue by dbaeumer** [Microsoft/vscode#24610]
- VSCode Version: 1.11.2 Smoke Test
- OS Version: Windows & NVDA 2017.1

NVDA is very silent on VSCode right now. Tabbing through button (side bar, search UI, ...) doesn't read anything. Nor does it read something in the editor itself. This used to work IMO.

**Comment by isidorn** [comment]
@dbaeumer can you please try with 1.11.1, 1.11.0 and 1.10.2 to see if this is a recent breakage, all of these builds can be found here https://vscode-builds.azurewebsites.net/stable

**Comment by dbaeumer** [comment]
After restarting VSCode I was not able to reproduce anymore. Will close.

**Comment by bpasero** [comment]
I cannot reproduce:

![image](https://cloud.githubusercontent.com/assets/900690/24966235/ab2a1f5c-1fa6-11e7-9d4d-fa97d9ed73cf.png)
